client/hostinfo_unix.cpp

```diff
@@ -2203,35 +2205,97 @@ long xss_idle() {
 }
 #endif // HAVE_XSS
 
+/**
+ * @brief Reads the timestamp specifically from /run/event_detect/last_active_time.dat
+ * using std::ifstream.
+ * @return int64_t Timestamp read from file, or -1 if the file cannot be opened/read/parsed.
+ */
+static int64_t ReadLastActiveTimeFromFile() { // Takes no arguments
+    const char* file_path = "/run/event_detect/last_active_time.dat";
+
+    std::ifstream infile(file_path); // Open the specific file
+
+    if (!infile.is_open()) {
+        msg_printf(NULL, MSG_INTERNAL_ERROR, "%s: Could not open file %s", __func__, file_path);
```

Comment:
We don't use `__func__`, `__line__` and `__file__` macroses.
Please remove them.